### DISCUSSION TITLE
Make Smaller/Make Larger size decrements/increments relative to screen size

### DISCUSSION THREAD

**Discussion by besfahbod** [rxhanson/Rectangle#642] [Ideas]
One of the features I miss from [ShiftIt](https://github.com/fikovnik/ShiftIt) is the ability to set the size of decrements/increments on Mark Smaller/Make Larger actions relative to the screen size, like in percentage of screen width/height.

I think a screen-size-relative approach is more useful when there's more than one screen frequently used, which can have very different dimensions.

From ShiftIt's Preferences:
![image](https://user-images.githubusercontent.com/45701775/142167683-ff287404-6984-4901-b3b2-92bb21e82ab5.png)

I'm looking at https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md and don't see any mention of this being possible. Have this option been discussed before on this project? Is it something you could consider adding?

**Comment by rxhanson** [comment]
Thanks for the idea, this has not been brought up before. I don’t have the bandwidth at the moment to add this in, but have no reason to reject a good PR for it. At first cut, I would only want this to be a terminal command config and maybe add it to the ui if there is enough demand.
